pandas/_libs/lib.pyx

```diff
@@ -678,6 +678,26 @@ def is_range_indexer(ndarray[int6432_t, ndim=1] left, Py_ssize_t n) -> bool:
     return True
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def is_range(ndarray[int6432_t, ndim=1] sequence, int64_t diff) -> bool:
```

Comment:
Does `diff`` mean `step`? If so can we stick with `step` as the name?

Reply:
Sure. Renamed to `step`